src/util/irep_ids.def

```diff
@@ -836,6 +836,7 @@ IREP_ID_TWO(C_java_generic_symbol, #java_generic_symbol)
 IREP_ID_TWO(generic_types, #generic_types)
 IREP_ID_TWO(implicit_generic_types, #implicit_generic_types)
 IREP_ID_TWO(type_variables, #type_variables)
+IREP_ID_TWO(java_lambda_method_handle_index, #lambda_method_handle_index)
```

Comment:
Comments by convention are called `ID_C_...`, though I wonder if this really should be a comment, if two `invokedynamic`s refer to two different method handles they probably should be not-equal

Reply:
This index is stored in the invoke dynamic instruction together with it's type which is of the form `Execute:(<arguments>)<lambda's_interface_name>` which I think should be just as unique as the index. So there's probably no harm in putting the index in comment but I'm not entirely sure about it. What would be the benefit of having it as a comment?

Reply:
I wasn't clear - it currently _is_ a comment (since the name starts with a `#` ), I agree with you - it should not be a comment.

Reply:
Oh I see, will change it.